rules/Transform/Rector/ArrayDimFetch/ArrayDimFetchToMethodCallRector.php

```diff
@@ -58,13 +58,13 @@ public function refactor(Node $node): ?MethodCall
             return null;
         }
 
-        foreach ($this->arrayDimFetchToMethodCalls as $arrayDimFetchToMethodCall) {
-            if (! $node->dim instanceof Node) {
-                return null;
-            }
+        if (! $node->dim instanceof Node) {
+            return null;
+        }
 
+        foreach ($this->arrayDimFetchToMethodCalls as $arrayDimFetchToMethodCall) {
             if (! $this->isObjectType($node->var, new ObjectType($arrayDimFetchToMethodCall->getClass()))) {
```

Comment:
@peterfox this can be improved, by move `new ObjectType()` in the configuration itself, so just call instead of re-create object, eg:

```diff
-new ArrayDimFetchToMethodCall('SomeClass', 'make')
+new ArrayDimFetchToMethodCall(new ObjectType('SomeClass'), 'make')
```

so only define `ObjectType()` once in the config ;), like what `ArgumentAdderRector` do:

https://github.com/rectorphp/rector-src/blob/d9b64c78e33ef461605ac08578c8159b01839c98/rules/Arguments/Rector/ClassMethod/ArgumentAdderRector.php#L86-L93